src/dataExplorer/components/FluxQueryBuilder.tsx

```diff
@@ -191,36 +161,30 @@ const FluxQueryBuilder: FC = () => {
                 ) : (
                   <Button
                     onClick={handleNewScript}
-                    text={
-                      isFlagEnabled('saveAsScript') ? 'New Script' : 'Clear'
-                    }
+                    text="New Script"
                     icon={IconFont.Plus_New}
                     testID="flux-query-builder--new-script"
                   />
                 )}
-                {isFlagEnabled('saveAsScript') && (
-                  <Button
-                    className="flux-query-builder__action-button"
-                    onClick={() => setOverlayType(OverlayType.OPEN)}
-                    text="Open"
-                    icon={IconFont.FolderOpen}
-                  />
-                )}
-                {isFlagEnabled('saveAsScript') && (
-                  <Button
-                    className="flux-query-builder__action-button"
-                    onClick={() => setOverlayType(OverlayType.SAVE)}
-                    status={
-                      hasChanged
-                        ? ComponentStatus.Default
-                        : ComponentStatus.Disabled
-                    }
-                    text="Save"
-                    testID="flux-query-builder--save-script"
-                    icon={IconFont.Save}
-                  />
-                )}
-                {isFlagEnabled('saveAsScript') && resource?.data?.id && (
+                <Button
+                  className="flux-query-builder__action-button"
+                  onClick={() => setOverlayType(OverlayType.OPEN)}
+                  text="Open"
+                  icon={IconFont.FolderOpen}
+                />
+                <Button
+                  className="flux-query-builder__action-button"
+                  onClick={() => setOverlayType(OverlayType.SAVE)}
+                  status={
+                    hasChanged
+                      ? ComponentStatus.Default
+                      : ComponentStatus.Disabled
+                  }
+                  text="Save"
+                  testID="flux-query-builder--save-script"
+                  icon={IconFont.SaveOutline}
```

Comment:
```suggestion
                  icon={IconFont.Save}
```
`SaveOutline` works fine in remocal but it's mapping pie chart in production and in tools. #6179 has more details about this issue. Let's use the solid icon `Save` for now until clockface fixes the icon file